### PR TITLE
Adds a bio for Amanda R.

### PR DESCRIPTION
**Pull request by gboone**
This is a bio for @ertzeid for the 18F website.

attn: @18F/blog @18F/content

### CODE REVIEW

File: _team/amanda-robinson.md [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
 github:
 last_name: Robinson
 name: amanda-robinson
-role:
-state:
-team:
+role: Developer
+state: DC
+team: Delivery
 twitter:
```
Comment (ertzeid) [inline _team/amanda-robinson.md]: Add @aertzeid (in whatever is the correct format) =)

File: _team/amanda-robinson.md [hunk 2]
```diff
@@ -1,12 +1,21 @@
 ---
-city:
+city: Washington
 first_name: Amanda
 full_name: Amanda Robinson
 github:
```
Comment (ertzeid) [inline _team/amanda-robinson.md]: github: @ertzeid

**Comment by ertzeid** [comment]
A couple additions, should you choose to add them (I'm happy to add afterward as well), but overall LGTM. Thanks!

**Comment by gboone** [comment]
No problem! This is ready to :shipit:

**Pull request merged**